pokemongo_bot/cell_workers/complete_tutorial.py

```diff
@@ -1,214 +0,0 @@
-import random
```

Comment:
IMO, we have "workers" that do specific functions in one place, then our "tasks" that have settings and choose which things we send to the "workers'. Say if a pokemon should be ignored, our task doesn't send it to the worker.